### PR TITLE
Improved code to Make white space normal the default for labels (#1zhk016)

### PR DESCRIPTION
**Pull request by azkyakhan**

### CODE REVIEW

**Comment by azkyakhan** [comment]
This node version issue is taken care by Disha.

File: src/theme/variables.css [hunk 1]
```diff
@@ -245,6 +245,10 @@ http://ionicframework.com/docs/theming/ */
   }
 }
 
+ion-item > ion-label {
+  white-space: normal !important;
```
Comment (dt2patel) [inline src/theme/variables.css]: does this not work without important? why not?

**Review by dt2patel — CHANGES_REQUESTED** [review]
See comments

File: src/theme/variables.css (hunk 1 above)
Comment (azkyakhan) [reply src/theme/variables.css]: Yes it is not over riding the white space without important.
label

Comment (azkyakhan) [reply src/theme/variables.css]: If I am putting style on this class, then its working.
Screenshot from 2022-02-09 00-01-19

Comment (dt2patel) [reply src/theme/variables.css]: we need a better solution than this because if we use this then we won't be able to over write this with a utility class.

**Comment by dt2patel** [comment]
Currently don't have a great way to do this. will try again later.

**Pull request closed without merging**